packages/shared/src/server/filterToPrisma.ts

```diff
@@ -136,9 +136,11 @@ export function tableColumnsToSqlFilter(
         : [Prisma.empty, Prisma.empty];
     const [funcPrisma1, funcPrisma2] =
       filter.type === "arrayOptions" && filter.operator === "none of"
-        ? [Prisma.raw("NOT ("), Prisma.raw(")")]
+        ? (table==="prompts" && filter.column === "Labels"
+          ? [Prisma.raw("name NOT IN (SELECT DISTINCT name FROM prompts p WHERE "), Prisma.raw(")")] 
```

Comment:
There is a subtle issue here in that the `project_id` filter is missing in this custom subquery. That means that if there is another project that has a prompt with the same name and label set, it will not show in the dashboard even though the prompt in the user project has that label not set.

Reply:
I checked a lot of approaches today
1. it would be difficult to handle in `promptRouter` as there can be combination of filters which are injected into the generate function together.
2. Additionally, the filter sql is returned from shared package's `filterToPrisma` as combined and joined. Hence, we can't break it down after calling the function.